src/main/java/com/idukbaduk/itseats/rider/service/RiderService.java

```diff
@@ -16,12 +15,18 @@
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.util.List;
+
 @Service
 @RequiredArgsConstructor
 public class RiderService {
 
     private final RiderRepository riderRepository;
     private final RiderAssignmentRepository riderAssignmentRepository;
+    private final OrderRepository orderRepository;
+
+    private final int DEFAULT_SEARCH_RADIUS_KM = 10;
```

Comment:
이 값이 변경되지 않는 상수라면, `private static final`로 선언해주시면 좋을 것 같아요. 
코드의 의도를 더 분명하게 드러낼 수 있을 것 같습니다.

일반적으로 상수는 클래스에서 가장 먼저 선언되기 때문에 선언 위치 수정도 부탁드립니다 🙇🏻‍♀️
```suggestion
    private static final int DEFAULT_SEARCH_RADIUS_KM = 10;
    
    private final RiderRepository riderRepository;
    private final RiderAssignmentRepository riderAssignmentRepository;
    private final OrderRepository orderRepository;
```